lib/auth/docker-compose.auth.yaml

```diff
@@ -35,8 +35,12 @@ services:
     cpu_shares: 512
     healthcheck:
       test: [ "CMD", "curl", "https://localhost:8443", "-k" ]
-      timeout: ${BENTOV2_HEALTHCHECK_TIMEOUT}
-      interval: ${BENTOV2_HEALTHCHECK_INTERVAL}
+      timeout: ${BENTO_HEALTHCHECK_TIMEOUT}
+      # interval: ${BENTO_HEALTHCHECK_INTERVAL}
+      # For Docker <25 and Compose <2.24.x, start_interval doesn't work - use a shorter interval for now
+      interval: 45s
+      start_period: ${BENTO_HEALTHCHECK_START_PERIOD}
+      start_interval: ${BENTO_HEALTHCHECK_START_INTERVAL}
```

Comment:
Confused by the comment, as this is a longer interval than `BENTO_HEALTHCHECK_START_INTERVAL` (5s)

Reply:
it's a bit confusing you're right - basically during the start period it checks every `start_interval` seconds, and then afterwards it checks every 45 seconds here (but ideally this would be even longer to avoid logspam, maybe every couple minutes). if Docker is <25/compose is <2.24, it instead ignores `start_interval` and so we need to check a bit more frequently than every couple minutes, otherwise anything waiting for a healthy status would take minutes to start.

once we ensure everyone and every instance has the right versions of Docker, we can remove this cruft in favour of a fast start interval and a slow interval after that.